src/hearings/store/actions/hearing-request.action.ts

```diff
@@ -8,10 +8,12 @@ export const UPDATE_HEARING_REQUEST = '[HEARING REQUEST] Update Hearing Request'
 export const UPDATE_HEARING_REQUEST_FAILURE = '[HEARING REQUEST] Update Hearing Request Failure';
 export const INITIALIZE_HEARING_REQUEST = '[HEARING REQUEST] Initialize Hearing Request';
 export const LOAD_HEARING_REQUEST = '[HEARING REQUEST] Load Hearing Request';
+export const LOAD_HEARING_REQUEST_FAILURE = '[HEARING REQUEST] Load Hearing Request Failure';
```

Comment:
is this needed anymore?

Reply:
I have removed this code which is not required.